speechbrain/nnet/RNN.py

```diff
@@ -102,6 +102,7 @@ def __init__(
         dropout=0.0,
         re_init=True,
         bidirectional=False,
+        padded_sequence_eval=True,
```

Comment:
This new variable is not described in the docstring

Reply:
Now this is actually not needing as parameter for the init, as we can use this as "shadow" flag to call inside the quantisation APIs..